README.md

```diff
@@ -160,6 +160,30 @@ async function test() {
   const payload = await verifyWithPromise(token)
   // => { a: 1, b: 2, c: 3, iat: 1579521212 }
 }
+
+// Verify using JWKS with key as a function and caching enabled (Uses get-jwks package)
+const { createVerifier } = require('fast-jwt')
+const buildGetJwks = require('get-jwks')
+
+// well known url of the token issuer
+// often encoded as the `iss` property of the token payload
+const domain = 'https://...'
```

Comment:
please use the `allowedDomains` option of `buildGetJwks`.

Reply:
I need the ```domain``` variable to pass it in ```getJwks.getPublicKey()``` when creating the verifier

Do you mean to also use ```allowedDomains``` option in ```buildGetJwks```, something like this?
```js
const getJwks = buildGetJwks({ allowedDomains: [domain]})
```